math/convertall/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+PortGroup           qt5 1.0
+
+github.setup        doug-101 ConvertAll 0.8.0 v
+github.tarball_from releases
+
+python.versions     34 35 36 37 38 39
+python.default_version     38
+
+qt5.min_version     5.4
+
+name                convertall
+version             0.8.0
+license             GPL-2+
+categories          math science
+platforms           darwin
+maintainers         {@khorton kilohotel.com:kevin01}
+description         Extremely flexible unit converter
+long_description    ConvertAll has a large database of units, and allows conversions \
+                    that use multiple units, e.g. convert from feet per decade to \
+                    nautical miles per fortnight.
+homepage            http://convertall.bellz.org
+
+checksums           rmd160  89fcc2aa9bad7ecc1de7bee4edea68fdff39706a \
+                    sha256  624c8a792b0bc7ff3776499c2c743b32273569efd0567615e570a7e739e8d521 \
+                    size    281055
+
+depends_lib-append  port:py${python.version}-pyqt5
+
+use_configure       no
+build.cmd           ${python.bin} install.py
+build.args          -p ${prefix} \
+                    -b ${destroot}
+
+destroot {}
+test {}
```

Comment:
you don't actually need to disable this phase -- if you haven't specifically enabled a test phase, then there is none by default.

Reply:
Removed line that disabled the test phase